src/beautifiers/php-cs-fixer.coffee

```diff
@@ -24,6 +24,7 @@ module.exports = class PHPCSFixer extends Beautifier
       "fix"
       "--rules=#{options.rules}" if options.rules
       "--config=#{configFile}" if configFile
+      "--allow-risky=#{options.allow_risky}" if options.allow_risky
```

Comment:
@Glavin001 I thought the if there is not passing this option as long as the allow_risky is not changed (not in the configuration file), after changed yes.

I know for sure that if not defined its set to no by default. So the general behaviour would be.
1. As long as it is not in the config file it will not be included (I am not sure about this)
2. If its set to no it will be included but nothing will change from the default behaviour
3. If it is set to yes then rules like symfony and so on (that are considered risky) may be applied